pom.xml

```diff
@@ -178,8 +177,9 @@
         <scala.compat.version>2.11</scala.compat.version>
         <scala.java8.compat.version>0.9.0</scala.java8.compat.version>
         <scala.version>2.11.12</scala.version>
-        <scalatest.maven.version>2.0.0</scalatest.maven.version>
-        <scalatest.version>3.2.2</scalatest.version>
+        <scalatest.maven.version>2.2.0</scalatest.maven.version>
+        <scalatest.version>3.2.14</scalatest.version>
+        <jacoco.version>0.8.8</jacoco.version>
```

Comment:
`jacoco` is a plugin, isn't it? Please move it to the plugins "section" and keep the alphabetical order.
(I know `scalatest.maven.version` is the same case - being fixed in another PR)

Reply:
- Property `jacoco.version` moved to correct alphabetical location.
- Property `jacoco.version` used in plugin definition. There was version string.
Changed in commit bfa34cae443c28e05ac6b42927b4bcf86b757d6a.